app/assets/scripts/components/home.js

```diff
@@ -119,7 +119,8 @@ var Home = React.createClass({
 
         <MapBoxMap
           query={this.props.query}
-          mapView={this.state.map.view}
+          params={this.props.params}
+          map={this.state.map}
```

Comment:
Nothing critical but it's usually a good idea to avoid using the word `map` on props, so it's not confused with the function `map()`